OpenKh.Engine/Parsers/PmoParser.cs

```diff
@@ -62,16 +50,63 @@ public PmoParser(Pmo pmo, float Scale)
 
                 MeshDescriptors.Add(currentMesh);
             }
+
+            if (pmo.header.SkeletonOffset != 0)
+            {
+                List<Matrix4x4> matrices = new List<Matrix4x4>();
+                List<Mdlx.Bone> skeleton = new List<Mdlx.Bone>();
+
+                foreach (Pmo.BoneData boneData in pmo.boneList)
+                {
+                    Matrix4x4 mtx = boneData.Transform;
+                    Matrix4x4 mtx_nd = Matrix4x4.Transpose(mtx);
+
+                    mtx_nd.M11 *= Scale;
+                    mtx_nd.M21 *= Scale;
+                    mtx_nd.M31 *= Scale;
+
+                    Vector3 loc;
+                    Quaternion quat;
+                    Vector3 scl;
+
+                    Matrix4x4.Decompose(mtx_nd, out scl, out quat, out loc);
```

Comment:
I can see you're decomposing the matrix but you are not using `scl`, `quat` and `loc`. Is this line of code needed?

Reply:
Just removed it!